### ISSUE TITLE
Unable to install Eazy Hax on Windows 10

### ISSUE THREAD

**Issue by secretclean** [Shakz76/Eazy-Hax-RetroPie-Toolkit#2]
Hello, how do I manually install these files? I've set everything to discoverable, but my Win10 laptop is unable to connect to my Pi 3 B+. The cmd window closes after a few seconds whenever I run your batches. Win10 also gives me a deprecation error when I try to manually connect with ssh via cmd. I'm only interested in the hard drive expansion, so I can load disc-based games on a hard drive and play them. Is there a way to install what you have manually via WinSCP?

Never mind, I connected manually with putty and ran the contents manually. SSH v.1 isn't supported in Windows 10.1, and I assume that's why your scripts didn't work. Anyway, the code doesn't recognize my hard drive as NTFS even though it is, both currently and natively.

**Comment by Shakz76** [comment]
Ok Windows 10 connection issue is fixed. Please download the new toolkit here
https://github.com/Shakz76/Eazy-Hax-RetroPie-Toolkit/raw/master/RetroPie%20Toolkit.zip
As for it not detecting the NTFS drive....does the hard drive have multiple NTFS partitions? If so that will break the script.

**Comment by GioThermal** [comment]
Still having issues with the windows command prompt exiting out after trying to install the eazyhax toolkit I used to have this working untill I decided to reinstall retropie on the same micro SD and have my regular images on it.Then installed the toolkit but it would still not work after downloading the new toolkit. :/ I need the toolkot for background music and for some retro pie tweaking.

**Comment by secretclean** [comment]
It should be vanilla RetroPie 4.4. I tried on fresh installs redownloaded from raspbian menu source twice (putty returns "Linux retropie 4.14.32v7-aufs #1 SMP Thu Apr 5 21:48:06 CEST 2018 armv7l
" when I log in). The only thing that might be different is doing the "Basic Install" in retropie setup to get the emulators to see if the emulators would register my roms. I have a Pi 3 B+ that I bought in a CanaKit package, not sure if that would make a difference. My drives only have one partition formatted to NTFS. Does it have to be MBR?

I also tried the new zip from the link you posted and had the same problem. The script that I ran above seemed to work, but neither the roms on SD or HDD registered. I ran it again from your newer build and it said "No hard drive detected." So I think it merged them and didn't work correctly after that.

**Comment by GioThermal** [comment]
I have had ssh enabled because I was connected to the pi via WinSCP

**Comment by Shakz76** [comment]
@secretclean Have you tried reinstalling Bonjour? Windows broke it with a patch a few patches back.... oh and my ssh question was directed @GioThermal  not you cuz I know winscp uses ssh ;-)
Can you post a screen shot or something because I cannot replicate this on my network. 
http://support.apple.com/downloads/DL999/en_US/BonjourPSSetup.exe

**Comment by secretclean** [comment]
I had to install the eazyhax menu manually with putty. I connected to my pi's ip address with putty.exe from the zip (the connection will be refused if your ssh is closed) and ran the script directly from https://github.com/Shakz76/Eazy-Hax-RetroPie-Toolkit/blob/master/cfg/Install%20Eazy%20Hax%20RetroPie%20Toolkit.sh. It kept saying 404 when I ran "curl https://github.com/Shakz76/Eazy-Hax-RetroPie-Toolkit/blob/master/cfg/Install%20Eazy%20Hax%20RetroPie%20Toolkit.sh | bash", so I just copied all the code, right-clicked to paste it in, then pressed enter. It all ran fine and rebooted, and now I have the eazyhax menu. Little workaround to the automation problem.

But yeah my roms are still blank when I run the expand script. They're on the sd and now the hard drive, and when I run the disable script they all work again. I'm trying this with a USB-powered hard drive, so maybe it's not drawing enough amps? But it's a 3 B+ with only 1 controller plugged in, and the drive is powered on and plays movies in Kodi fine. I'll have to get a hard drive with power source and report back.
